Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml

```diff
@@ -3,324 +3,324 @@ commonfields:
   id: Intezer v2
   version: -1
 configuration:
-  - display: API Key
-    name: APIKey
-    required: true
-    type: 4
-  - display: Intezer Analyze Base URL
-    name: AnalyzeBaseURL
-    defaultvalue: https://analyze.intezer.com/api/
-    type: 0
-    required: false
-    additionalinfo: The API address to intezer Analyze - i.e. http(s)://<address>/api/
-  - display: Use system proxy settings
-    name: proxy
-    required: false
-    type: 8
-  - display: Trust any certificate (not secure)
-    name: insecure
-    required: false
-    type: 8
+- display: API Key
+  name: APIKey
+  required: true
+  type: 4
+- display: Intezer Analyze Base URL
+  name: AnalyzeBaseURL
+  defaultvalue: https://analyze.intezer.com/api/
+  type: 0
+  required: false
```

Comment:
While this is not part of the change, are you sure you want the url to be allowed to remain empty?

Reply:
If I understand correctly if it's not set, the default value is used, which is the expected behavior

Reply:
The default value is what we see when configuring the integration.
If it is erased, since the param is not required It will take the value to be an empty string.
But making this param required will require breaking changes, and will not be backward compatible so let's leave it for now.
Another option that will not break backward- you can handle it as part of the code in the main function: if the value is an empty string, change t to the hardcoded string you want. But I leave it for you to decide.

Reply:
Let's leave as is. I might change it when we will completely remove the d2-related playbook since it will be a breaking change.